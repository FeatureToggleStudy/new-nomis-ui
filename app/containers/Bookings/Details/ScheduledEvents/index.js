import ScheduledEvents from './ScheduledEvents'

export default ScheduledEvents
