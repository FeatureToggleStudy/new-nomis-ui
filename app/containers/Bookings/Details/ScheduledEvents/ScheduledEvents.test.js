import React from 'react'
import { shallow } from 'enzyme'
import { Map, List } from 'immutable'
import { ScheduledEvents } from './ScheduledEvents'

const events = List([
  Map({
    date: '2019-02-27T15:40:17.317Z',
    morningActivities: List([]),
    afternoonActivities: List([]),
    eveningDuties: List([]),
  }),
  Map({
    date: '2019-02-28T15:40:17.317Z',
    morningActivities: List([]),
    afternoonActivities: List([]),
    eveningDuties: List([]),
  }),
  Map({
    date: '2019-03-01T15:40:17.317Z',
    morningActivities: List([
      Map({
        type: 'Careers Adviser Appointment',
        comment: "I'm a recurring bulk appointment.",
        shortComment: "I'm a recurring bulk appointment.",
        startTime: '2019-03-01T10:00:00',
        endTime: '2019-03-01T10:10:00',
        cancelled: false,
        eventStatus: 'SCH',
      }),
      Map({
        type: 'Another appointment',
        comment: "I'm another appointment.",
        shortComment: "I'm another appointment.",
        startTime: '2019-03-01T11:00:00',
        endTime: '2019-03-01T12:00:00',
        cancelled: true,
        eventStatus: 'SCH',
      }),
    ]),
    afternoonActivities: List([]),
    eveningDuties: List([]),
  }),
  Map({
    date: '2019-03-02T15:40:17.317Z',
    morningActivities: List([]),
    afternoonActivities: List([]),
    eveningDuties: List([]),
  }),
  Map({
    date: '2019-03-03T15:40:17.317Z',
    morningActivities: List([]),
    afternoonActivities: List([]),
    eveningDuties: List([]),
  }),
  Map({
    date: '2019-03-04T15:40:17.317Z',
    morningActivities: List([]),
    afternoonActivities: List([]),
    eveningDuties: List([]),
  }),
  Map({
    date: '2019-03-05T15:40:17.317Z',
    morningActivities: List([]),
    afternoonActivities: List([
      Map({
        type: 'Another appointment',
        comment: "I'm another appointment.",
        shortComment: "I'm another appointment.",
        startTime: '2019-03-01T11:00:00',
        endTime: '2019-03-01T12:00:00',
        cancelled: true,
        eventStatus: 'SCH',
      }),
    ]),
    eveningDuties: List([]),
  }),
])

describe('<Scheduled />', () => {
  const props = {
    offenderNo: 'A12345',
    scheduledEvents: events,
    offenderDetails: {
      firstName: 'Test',
      lastName: 'User',
    },
    currentFilter: Map({ thisWeek: true }),

    loadThisWeeksScheduledEvents: jest.fn(),
    loadNextWeeksScheduledEvents: jest.fn(),
    loadBookingDetails: jest.fn(),
  }

  it('renders correctly', () => {
    const wrapper = shallow(<ScheduledEvents {...props} />)

    console.log(wrapper.debug())
  })

  it('should pass the correct offender name to the page title', () => {
    const wrapper = shallow(<ScheduledEvents {...props} />)

    expect(wrapper.find('withRouter(Connect(Page))').prop('title')).toEqual('Schedule for Test User')
  })
})
