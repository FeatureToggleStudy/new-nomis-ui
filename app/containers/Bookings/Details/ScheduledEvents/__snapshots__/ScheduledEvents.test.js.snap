// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Scheduled /> should match the default snapshot 1`] = `
Array [
  <div
    className="sc-bwzfXH kkYVmt"
  >
    <div
      className="css-2n4gqn"
    >
      <ul
        className="css-1me48lg"
      >
        <li
          className="css-122wdxf"
        >
          <span
            location={
              Object {
                "hash": "",
                "key": "testKey",
                "pathname": "/",
                "search": "",
              }
            }
            match={
              Object {
                "isExact": true,
                "params": Object {},
                "path": "/",
                "url": "/",
              }
            }
          >
            Home
          </span>
        </li>
      </ul>
    </div>
  </div>,
  <div
    className="sc-EHOje gGvWDh"
  >
    <div
      className="sc-bZQynM gJmhNi"
    >
      <h1
        className="sc-bdVaJa eLwxMn"
        data-qa="page-heading-text"
      >
        Schedule for Test User
      </h1>
      <div>
        <div
          className="sc-gzVnrw kmmJFT"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="link"
          tabIndex={0}
        >
          <span>
            Print this page
          </span>
        </div>
      </div>
    </div>
    <div
      className="page-content"
    >
      <div
        className="css-1twtykl"
      >
        <label
          className="eghbvew0 css-p3popv-Label-StyledHoc eqinktp0"
        >
          <input
            checked="checked"
            className="css-1cykcm5-Input eqinktp1"
            id="thisWeek"
            name="radio-inline-group"
            onChange={[Function]}
            onClick={[Function]}
            type="radio"
            value="Yes"
          />
          <span
            className="css-11gu4gd-LabelText eqinktp2"
          >
            This week
          </span>
        </label>
        <label
          className="eghbvew0 css-p3popv-Label-StyledHoc eqinktp0"
        >
          <input
            checked={undefined}
            className="css-1cykcm5-Input eqinktp1"
            id="nextWeek"
            name="radio-inline-group"
            onChange={[Function]}
            onClick={[Function]}
            type="radio"
            value="Yes"
          />
          <span
            className="css-11gu4gd-LabelText eqinktp2"
          >
            Next week
          </span>
        </label>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Wednesday
           
          <span>
            02/27/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Thursday
           
          <span>
            02/28/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Friday
           
          <span>
            03/01/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              <div
                className="css-8fnfo7"
              >
                <span
                  className="whereabouts-startTime"
                >
                  10:00
                </span>
                 
                - 
                <span
                  className="whereabouts-endTime"
                >
                   
                  10:10
                   
                </span>
                <div
                  className="css-g90fw7"
                >
                  Careers Adviser Appointment
                </div>
                I'm a recurring bulk appointment.
              </div>
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Saturday
           
          <span>
            03/02/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Sunday
           
          <span>
            03/03/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Monday
           
          <span>
            03/04/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Tuesday
           
          <span>
            03/05/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              <div
                className="css-8fnfo7"
              >
                <span
                  className="whereabouts-startTime"
                >
                  13:00
                </span>
                 
                - 
                <span
                  className="whereabouts-endTime"
                >
                   
                  14:00
                   
                </span>
                <div
                  className="css-g90fw7"
                >
                  Another appointment
                </div>
                I'm another appointment.
              </div>
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="sc-gzVnrw kmmJFT"
    onClick={[Function]}
    onKeyDown={[Function]}
    role="link"
    tabIndex={0}
  >
    <span>
      Print this page
    </span>
  </div>,
]
`;

exports[`<Scheduled /> should match the snapshot with a cancelled event 1`] = `
Array [
  <div
    className="sc-bwzfXH kkYVmt"
  >
    <div
      className="css-2n4gqn"
    >
      <ul
        className="css-1me48lg"
      >
        <li
          className="css-122wdxf"
        >
          <span
            location={
              Object {
                "hash": "",
                "key": "testKey",
                "pathname": "/",
                "search": "",
              }
            }
            match={
              Object {
                "isExact": true,
                "params": Object {},
                "path": "/",
                "url": "/",
              }
            }
          >
            Home
          </span>
        </li>
      </ul>
    </div>
  </div>,
  <div
    className="sc-EHOje gGvWDh"
  >
    <div
      className="sc-bZQynM gJmhNi"
    >
      <h1
        className="sc-bdVaJa eLwxMn"
        data-qa="page-heading-text"
      >
        Schedule for Test User
      </h1>
      <div>
        <div
          className="sc-gzVnrw kmmJFT"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="link"
          tabIndex={0}
        >
          <span>
            Print this page
          </span>
        </div>
      </div>
    </div>
    <div
      className="page-content"
    >
      <div
        className="css-1twtykl"
      >
        <label
          className="eghbvew0 css-p3popv-Label-StyledHoc eqinktp0"
        >
          <input
            checked="checked"
            className="css-1cykcm5-Input eqinktp1"
            id="thisWeek"
            name="radio-inline-group"
            onChange={[Function]}
            onClick={[Function]}
            type="radio"
            value="Yes"
          />
          <span
            className="css-11gu4gd-LabelText eqinktp2"
          >
            This week
          </span>
        </label>
        <label
          className="eghbvew0 css-p3popv-Label-StyledHoc eqinktp0"
        >
          <input
            checked={undefined}
            className="css-1cykcm5-Input eqinktp1"
            id="nextWeek"
            name="radio-inline-group"
            onChange={[Function]}
            onClick={[Function]}
            type="radio"
            value="Yes"
          />
          <span
            className="css-11gu4gd-LabelText eqinktp2"
          >
            Next week
          </span>
        </label>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Wednesday
           
          <span>
            02/27/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Thursday
           
          <span>
            02/28/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Friday
           
          <span>
            03/01/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              <div
                className="css-8fnfo7"
              >
                <span
                  className="whereabouts-startTime"
                >
                  10:00
                </span>
                 
                - 
                <span
                  className="whereabouts-endTime"
                >
                   
                  10:10
                   
                </span>
                <div
                  className="css-g90fw7"
                >
                  Careers Adviser Appointment
                </div>
                I'm a recurring bulk appointment.
              </div>
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Saturday
           
          <span>
            03/02/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Sunday
           
          <span>
            03/03/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              <div
                className="css-8fnfo7"
              >
                <span
                  className="whereabouts-startTime"
                >
                  11:00
                </span>
                 
                - 
                <span
                  className="whereabouts-endTime"
                >
                   
                  11:30
                   
                </span>
                <div
                  className="css-g90fw7"
                >
                  Another appointment
                </div>
                I'm another appointment.
                <span
                  className="cancelled"
                >
                   (cancelled)
                </span>
              </div>
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Monday
           
          <span>
            03/04/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-1tuknv3"
      >
        <h2
          className="sc-bdVaJa jAqeHE"
        >
          Tuesday
           
          <span>
            03/05/2019
          </span>
        </h2>
        <div
          className="css-1yvuvcl"
        >
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Morning (AM)
            </h3>
            <div
              className="css-17o0f5y"
              value="morning"
            >
              No scheduled events.
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Afternoon (PM)
            </h3>
            <div
              className="css-10kxiuv"
              value="afternoon"
            >
              <div
                className="css-8fnfo7"
              >
                <span
                  className="whereabouts-startTime"
                >
                  13:00
                </span>
                 
                - 
                <span
                  className="whereabouts-endTime"
                >
                   
                  14:00
                   
                </span>
                <div
                  className="css-g90fw7"
                >
                  Another appointment
                </div>
                I'm another appointment.
              </div>
            </div>
          </div>
          <div
            className="css-wwpffz"
          >
            <h3
              className="sc-bdVaJa dbbMZC"
            >
              Evening Duty (ED)
            </h3>
            <div
              className="css-18nhmyv"
              value="evening"
            >
              No scheduled events.
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="sc-gzVnrw kmmJFT"
    onClick={[Function]}
    onKeyDown={[Function]}
    role="link"
    tabIndex={0}
  >
    <span>
      Print this page
    </span>
  </div>,
]
`;
